src/scenes/ProgressReports/Detail/ProgressReportDrawer/ProgressReportDrawer.tsx

```diff
@@ -1,29 +1,42 @@
+import { useQuery } from '@apollo/client';
```

Comment:
This will create major conflicts for the branches that are **based** on this PRs branch, just FYI. I don't feel comfortable making this change in this PR. I can make this change in the Progress step PR